src/core/dev/allocator.cpp

```diff
@@ -255,36 +255,91 @@ xlio_registrator::~xlio_registrator()
 }
 
 bool xlio_registrator::register_memory(void *data, size_t size)
+{
+    bool rc = true;
+#ifdef DEFINED_DPCP_PATH_RX_OR_TX
```

Comment:
where is place with `DEFINED_DPCP_PATH_RX_AND_TX` , `DEFINED_DPCP_PATH_RX_OR_TX` etc definition.
I see just `DEFINED_DPCP_PATH_RX` and `DEFINED_DPCP_PATH_TX`

Reply:
util/vtypes.h

Reply:
Maybe it will not be needed if we follow the second comment.

Reply:
Names will be renamed to DEFIEND_DPCP_ONLY (for TX and RX), DEFINED_DPCP_MIX, DEFINED_NO_DPCP (Only DOCA for easier reading)

Reply:
Renamed DEFINED_DPCP_PATH_RX_AND_TX/DEFINED_DPCP_PATH_RX_OR_TX to DEFINED_DPCP_PATH_ONLY and DEFINED_DPCP_PATH_ANY respectively.